spring-cloud-dataflow-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/ApiDocumentation.java

```diff
@@ -93,7 +93,7 @@ public void index() throws Exception {
 					linkWithRel("runtime/apps/instances").description("Provides the status for app instances"),
 					linkWithRel("tasks/definitions").description("Provides the task definition resource"),
 					linkWithRel("tasks/definitions/definition").description("Provides details for a specific task definition"),
-					linkWithRel("tasks/composed-definitions/compose").description("Requests the creation of a composed task definition"),
```

Comment:
you'll obviously need to change it at several places. But you need to distinguish between **the name of the relation** and the **URL**